Core/Core/Serialisation/BaseObjectDeserializerV2.cs

```diff
@@ -123,7 +123,7 @@ private List<(string, int)> GetClosures(string rootObjectJson)
   private object DeserializeTransportObjectProxy(string objectJson)
   {
     // Try background work
-    Task<object> bgResult = WorkerThreads.TryStartTask(WorkerThreadTaskType.Deserialize, objectJson);
+    Task<object> bgResult = WorkerThreads.TryStartTask(WorkerThreadTaskType.Deserialize, objectJson); //BUG: Because we don't guarantee this task will ever be awaited, this may lead to unobserved exceptions!
     if (bgResult != null)
```

Comment:
Do we want to track things I don't want to fix? 🤣 

It's not really a bug, it won't cause any functional problems for users, but it does mean more noise in seq, since this will become a Fatal log.

I've been investigating some similar issues in other parts of our code, so potentially, once I have a clearer understanding of the sources for these errors, I'll track the ones that remian.